### PR TITLE
[sqlite-orm] Update SQLite ORM to 1.8

### PR DESCRIPTION
**Pull request by trueqbit**
**Describe the pull request**

- #### What does your PR fix?
  Updates SQLite ORM [sqlite-orm] to v1.8.1.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  CI baseline updated.
  tested triplets: sqlite-orm[core,example,test]:x64-windows, sqlite-orm[core,example,test]:x86-windows, sqlite-orm[core,example]:x64-windows-v140

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  Yes

### CODE REVIEW

**Review by LilyWangLL — DISMISSED** [review]

**Comment by trueqbit** [comment]
Waiting for https://github.com/fnc12/sqlite_orm/pull/1125

**Comment by trueqbit** [comment]
> _This is a new experimental fast check for PR issues. Please let us know if this bot is helpful!_
> PRs must add only one version and must not modify any published versions
> 
> When making any changes to a library, the version or port-version in `vcpkg.json` or `CONTROL` must be modified.
> 
> ```
> error: checked-in files for sqlite-orm have changed but the version was not updated
> version: 1.8.1
> old SHA: a120f5c5e79bdce818865ccfe2883c8e5296aaeb
> new SHA: cbb80bd62fb69f81d00cc551f1033b31df52ca2d
> Did you remember to update the version or port version?
> Use --overwrite-version to bypass this check
> ***No files were updated***
> ```

I don't quite understand the problem. Is it the two commits in the PR that change the version? Do I need to create another PR that combines all the changes?

**Comment by LilyWangLL** [comment]
> > _This is a new experimental fast check for PR issues. Please let us know if this bot is helpful!_
> > PRs must add only one version and must not modify any published versions
> > When making any changes to a library, the version or port-version in `vcpkg.json` or `CONTROL` must be modified.
> > ```
> > error: checked-in files for sqlite-orm have changed but the version was not updated
> > version: 1.8.1
> > old SHA: a120f5c5e79bdce818865ccfe2883c8e5296aaeb
> > new SHA: cbb80bd62fb69f81d00cc551f1033b31df52ca2d
> > Did you remember to update the version or port version?
> > Use --overwrite-version to bypass this check
> > ***No files were updated***
> > ```
> 
> I don't quite understand the problem. Is it the two commits in the PR that change the version? Do I need to create another PR that combines all the changes?

You need change SHA in file **[VCPKG_ROOT]/versions/s-/sqlite-orm.json**.
```
 old SHA: a120f5c5e79bdce818865ccfe2883c8e5296aaeb
 new SHA: cbb80bd62fb69f81d00cc551f1033b31df52ca2d
```
image

File: versions/s-/sqlite-orm.json [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+    {
+      "git-tree": "cbb80bd62fb69f81d00cc551f1033b31df52ca2d",
+      "version": "1.8.1",
+      "port-version": 0
+    },
+    {
+      "git-tree": "a264108d68b73d47e82f31984be134b7c0d73b85",
+      "version": "1.8.0",
+      "port-version": 0
+    },
```
Comment (LilyWangLL) [inline versions/s-/sqlite-orm.json:12]:
```suggestion
```

Comment (LilyWangLL) [reply versions/s-/sqlite-orm.json:12]: This need be removed.

**Review by LilyWangLL — CHANGES_REQUESTED** [review]

**Review by LilyWangLL — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**